docker-compose.yaml

```diff
@@ -33,6 +33,48 @@ services:
     logging: *default-logging
     container_name: airbyte-scheduler
     restart: unless-stopped
+    environment:
+      - WEBAPP_URL=${WEBAPP_URL}
+      - DATABASE_USER=${DATABASE_USER}
+      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
+      - DATABASE_URL=${DATABASE_URL}
+      - CONFIG_DATABASE_USER=${CONFIG_DATABASE_USER:-}
+      - CONFIG_DATABASE_PASSWORD=${CONFIG_DATABASE_PASSWORD:-}
+      - CONFIG_DATABASE_URL=${CONFIG_DATABASE_URL:-}
+      - RUN_DATABASE_MIGRATION_ON_STARTUP=${RUN_DATABASE_MIGRATION_ON_STARTUP}
+      - WORKSPACE_ROOT=${WORKSPACE_ROOT}
+      - WORKSPACE_DOCKER_MOUNT=${WORKSPACE_DOCKER_MOUNT}
+      - LOCAL_ROOT=${LOCAL_ROOT}
+      - LOCAL_DOCKER_MOUNT=${LOCAL_DOCKER_MOUNT}
+      - CONFIG_ROOT=${CONFIG_ROOT}
+      - TRACKING_STRATEGY=${TRACKING_STRATEGY}
+      - AIRBYTE_VERSION=${VERSION}
+      - AIRBYTE_ROLE=${AIRBYTE_ROLE:-}
+      - TEMPORAL_HOST=${TEMPORAL_HOST}
+      - WORKER_ENVIRONMENT=${WORKER_ENVIRONMENT}
+      - S3_LOG_BUCKET=${S3_LOG_BUCKET}
+      - S3_LOG_BUCKET_REGION=${S3_LOG_BUCKET_REGION}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - GCP_STORAGE_BUCKET=${GCP_STORAGE_BUCKET}
+      - LOG_LEVEL=${LOG_LEVEL}
+      - SUBMITTER_NUM_THREADS=${SUBMITTER_NUM_THREADS}
+      - RESOURCE_CPU_REQUEST=${RESOURCE_CPU_REQUEST}
+      - RESOURCE_CPU_LIMIT=${RESOURCE_CPU_LIMIT}
+      - RESOURCE_MEMORY_REQUEST=${RESOURCE_MEMORY_REQUEST}
+      - RESOURCE_MEMORY_LIMIT=${RESOURCE_MEMORY_LIMIT}
+      - MAX_SYNC_JOB_ATTEMPTS=${MAX_SYNC_JOB_ATTEMPTS}
+      - MAX_SYNC_TIMEOUT_DAYS=${MAX_SYNC_TIMEOUT_DAYS}
+      - INTERNAL_API_HOST=${INTERNAL_API_HOST}
+    volumes:
+      - workspace:${WORKSPACE_ROOT}
+      - ${LOCAL_ROOT}:${LOCAL_ROOT}
+      - data:${CONFIG_ROOT}
+  worker:
```

Comment:
should we care that we are adding another container for the docker-compose setup? even if we are fine with doing it now, is it in general preferable for us to minimize number of containers in the future?

Reply:
This is unfortunate, but I think it's a necessary evil. The alternative is to have the main containers mean different things on different platforms. I think worker is core enough that it's worth splitting.